server/repository/repository_test.go

```diff
@@ -4,10 +4,10 @@ import "testing"
 
 func TestRepoURLToSecretName(t *testing.T) {
 	tables := map[string]string{
-		"git://git@github.com:argoproj/ARGO-cd.git": "repo-argo-cd-83273445",
-		"https://github.com/argoproj/ARGO-cd":       "repo-argo-cd-1890113693",
-		"https://github.com/argoproj/argo-cd":       "repo-argo-cd-42374749",
-		"ssh://git@github.com/argoproj/argo-cd.git": "repo-argo-cd-3569564120",
+		"git://git@github.com:argoproj/ARGO-cd.git": "repo-argo-cd-593837413",
+		"https://github.com/argoproj/ARGO-cd":       "repo-argo-cd-821842295",
+		"https://github.com/argoproj/argo-cd":       "repo-argo-cd-821842295",
```

Comment:
Can we add one where "https://github.com/argoproj/argo-cd.git" also maps to "repo-argo-cd-821842295"